src/ducks/transform/certificates.ts

```diff
@@ -172,3 +174,195 @@ export function transformCertificateUploadModelToDto(upload: CertificateUploadMo
 export function transformCertificateComplianceCheckModelToDto(check: CertificateComplianceCheckModel): CertificateComplianceCheckDto {
     return { ...check };
 }
+
+// fix typo
+export function transformCertifacetObjectToNodesAndEdges(certificate: CertificateDetailResponseModel, users: UserDto[]) {

```

Comment:
users should be UserModel[]

Reply:
the selector of this state is giving us an object of type UserDto, which we are accessing inside the hook..

Reply:
it's     users: UserResponseModel[];